addonlibs/comm/MessageSys/model/MessageSys.ovm

```diff
@@ -1,6 +1,24 @@
 #include "ov.ovm"
 #include "ksapi.ovm"
 
+/*
+	*
+	*   Chair of Process Control Engineering
```

Comment:
You are right. We will fix this. This was only the first step to introduce the new license and to show Simon how PRs work.
There are even more License Header to change in the source files ;)